frontend/src/pages/PersonalPage.jsx

```diff
@@ -1,12 +1,268 @@
 import Template from "../components/template";
+import ModalHeader from "../components/modalHeader.jsx";
+import ModalLeftWindow from "../components/modalLeftWindow";
+import Button from "../components/button";
+import { useState, React, useEffect } from "react";
+import AddFriendWindow from "../components/addFriendWindow";
+import "../css/personalPage.css";
+import UploadNotesWindow from "../components/uploadNotesWindow";
+import { Link } from "react-router-dom";
+import PDFthumbnail from "../components/PDFthumbnail";
 
 function PersonalPage(props) {
+    const [buttonAddFriend, setButtonAddFriend] = useState(false);
+    const [buttonGroup, setButtonGroup] = useState(false);
+    const [personalProfile, setButtonPersonalProfile] = useState(false);
```

Comment:
For consistency, this should be ``buttonPersonalProfile, setButtonPersonalProfile``

Reply:
Changed it.